src/js/components/Breadcrumb.js

```diff
@@ -46,6 +72,12 @@ class Breadcrumb extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
+    let {buildCrumbCustomParams} = this.props;
+
+    if (!DeepEqual(nextProps.buildCrumbCustomParams, buildCrumbCustomParams)) {
```

Comment:
Don't think we need this right? The last part of this method already checks if props have changed